### PR TITLE
1.13.0 Release Docs

### PR DESCRIPTION
**Pull request by resmaa**
1. Release Notes
2. MaaS
3. Packs and CLI updates

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/435#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: packages/docs/content/04-clusters/1-new-clusters.md [hunk 1]
```diff
@@ -2550,4 +2550,274 @@ New worker pools may be added if it desired to customize certain worker nodes to
...
+|  API Endpoint |  API Endpoint of the gateway   |
+| API Key| 
+
+Validate the above MaaS credentials to create your MaaS cloud account.
+
+## Creating a MaaS Cluster
+
+The following steps need to be performed to provision a new MaaS cluster:-
+
+* Provide basic cluster information like name, description, and tags. Tags are currently not propagated to the VMs deployed on the cloud/data center environments.
```
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2743]: Change all the VMs to Machines as in MaaS it is termed as machines and not VM's

File: packages/docs/content/04-clusters/1-new-clusters.md [hunk 2]
```diff
@@ -2550,4 +2550,274 @@ New worker pools may be added if it desired to customize certain worker nodes to
...
+   * Domain 
+  
+* Configure the master and worker node pools. A master and a worker node pool are configured by default.
+    * Name - A descriptive name for the node pool.
+    * Size - Number of nodes to be provisioned for the node pool. For the master pool, this number can be 1, 3, 5, etc.
+    * Allow worker capability (master pool) - To workloads to be provisioned on master nodes.
+    * Resource Pool	
+    * Availability zones 
+    * Minimum CPU
+    * Disk - Storage disk size in GB to be attached to the node.
```
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2763]: Minimum Disk Storage

**Review by arvind-sys — CHANGES_REQUESTED** [review]

**Pull request merged**